charts/pega/README.md

```diff
@@ -238,6 +238,7 @@ Specify the `ingress` yaml block to expose a Pega tier to access from outside Ku
 Parameter | Description
 ---       | ---
 `domain`  | Specify a domain on your network in which you create an ingress to the load balancer.
+`appContextPath`  | Specify the application context path to be used to access applications on a specific tier.  If not specified, /prweb will used. 
```

Comment:
I'm assuming that you don't need to specify the leading '/' ?

Reply:
Updated description to remove leading slash.  It is not needed.